api_test.go

```diff
@@ -367,7 +373,7 @@ func getAPI(url, password string, register bool) *CacophonyAPI {
 }
 
 func TestFileDownload(t *testing.T) {
-	Fs = afero.NewMemMapFs()
+	require.NoError(t, newFs())
```

Comment:
If you pass `t` to `newFs` you can avoid all the `require.NoError` calls in each test (`newFs` can check for errors). This is a common pattern for test helper functions in Go.